server/routes/components.ts

```diff
@@ -50,8 +63,31 @@ export default function routes({ serviceCatalogueService, redisService }: Servic
       alerts_slack_channel: alertsSlackChannel,
       github_enforce_admins_enabled: component.github_enforce_admins_enabled,
       standardsCompliance: component.standards_compliance,
+      alerts: [] as DisplayAlert[],
     }
 
+    let alerts: DisplayAlert[] = []
+    const alertManagerUrl = process.env.ALERTMANAGER_URL
```

Comment:
You'd pull the alert manager URL into `/server/config.ts` like all the other env vars then in here you'd just include config and pull it out from that 👍